http_check/conf.yaml.example

```diff
@@ -95,11 +95,14 @@ instances:
     # left in the certificate, and alternatively raise a critical
     # warning if the certificate is y days from the expiration date.
     # The SSL certificate will always be validated for this additional
-    # service check regardless of the value of disable_ssl_validation
+    # service check regardless of the value of disable_ssl_validation.
+    # The Agent uses hostname by default to validate the SSL cert, but if
```

Comment:
nit: IMO we could be a bit more explicit about what this does, maybe something like `By default, for the expiration check, the Agent validates the SSL certificate hostname against the host of the provided url. You can override that hostname with ssl_server_name.`? What do you think?

Reply:
Sounds good to me!